src/EventHandler/EventInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\EventHandler;
+
+interface EventInterface
+{
+    public function getArray():array;
```

Comment:
This is not descriptive enough. What is it actually getting?

Reply:
I agree. An array is a type and this is just not describing the kind of data it accesses.
So this could for example  be called `getMetaData` or `getHeaders` depending on the actual usage.